src/imgtools/_deprecated/ops/ops.py

```
@@ -0,0 +1,495 @@
+from typing import Any, Dict, List, Optional, Sequence, Tuple, TypeVar, Union
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import

(F401)